### PR TITLE
add host port to intranet link

### PR DESCRIPTION
**Pull request by jpuck**
For those who run a custom port using `HOST_PORT_HTTPD` instead of 80, this fixes the link on vhosts.php

### CODE REVIEW

**Comment by jpuck** [comment]
Because the tests were [already failing][1] on master, I doubt my changes made anything worse.

[1]:https://travis-ci.org/cytopia/devilbox/builds

File: .devilbox/www/htdocs/vhosts.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ function updateStatus(vhost) {
 						} else {
 							el_valid.className += ' bg-success';
 							el_valid.innerHTML = 'OK';
-							el_href.innerHTML = '<a target="_blank" href="http://'+vhost+'.<?php echo $Docker->getTld();?>">'+vhost+'.<?php echo $Docker->getTld();?></a>';
+							el_href.innerHTML = '<a target="_blank" href="http://'+vhost+'.<?php echo $Docker->getTld().$Docker->getPort();?>">'+vhost+'.<?php echo $Docker->getTld();?></a>';
```
Comment (cytopia) [inline .devilbox/www/htdocs/vhosts.php]: I'd say, you also add the port to the display itself. `<a href="...:8080">also here I would say</a>`:
```
href="http://'+vhost+'.<?php echo $Docker->getTld().$Docker->getPort();?>">'+vhost+'.<?php echo $Docker->getTld().$Docker->getPort();?></a>';
```
What do you think?

Comment (jpuck) [inline .devilbox/www/htdocs/vhosts.php]: @cytopia agreed. Pushed update

**Comment by cytopia** [comment]
Awesome, thanks.

**Pull request merged**